src/deluge/gui/ui_timer_manager.h

```diff
@@ -34,13 +34,11 @@
 #define TIMER_READ_INPUTS 12
 #define TIMER_BATT_LED_BLINK 13
 #define TIMER_GRAPHICS_ROUTINE 14
-#define TIMER_AUTOMATION_VIEW 15
```

Comment:
removed TIMER_AUTOMATION_VIEW as its no longer being used and renumbered the timer enum's below it accordingly.